manim/camera/mapping_camera.py

```diff
@@ -20,22 +22,22 @@ class MappingCamera(Camera):
 
     def __init__(
         self,
-        mapping_func=lambda p: p,
-        min_num_curves=50,
-        allow_object_intrusion=False,
+        mapping_func: Callable = lambda p: p,
+        min_num_curves: int = 50,
+        allow_object_intrusion: bool = False,
         **kwargs
     ):
         self.mapping_func = mapping_func
         self.min_num_curves = min_num_curves
         self.allow_object_intrusion = allow_object_intrusion
         Camera.__init__(self, **kwargs)
 
-    def points_to_pixel_coords(self, points):
+    def points_to_pixel_coords(self, points: int):
```

Comment:
Issue : I don't think it's an int, but rather an np array

Reply:
your right. I'll fix that.